src/__tests__/components/articleComponent/__test__/articleComponent.test.js

```diff
@@ -36,4 +36,22 @@ describe("Article component test", () => {
     const wrapper = shallow(<ArticleComponent {...props} />);
     expect(wrapper).toMatchSnapshot();
   });
+
+  it("should render without crashing", () => {
+    const props = {
+      article: {
+        title: "test title",
+        image: null,
+        tagList: [],
```

Comment:
I dont think you should have tagList as its yet to be implemented

Reply:
Okay let me remove that

Reply:
No its fine its needed just went through it now you remove it and tests will fail

Reply:
> I dont think you should have tagList as its yet to be implemented

Okay so if I remove that, the tests all fail because apparently it was included somewhere in the article component and is being used in other tests as well
![image](https://user-images.githubusercontent.com/42435346/56560181-654f8000-65ac-11e9-8f3e-8ee4892b7220.png)